build.sbt

```diff
@@ -23,7 +23,7 @@ lazy val commonSettings = Seq(
   organization := "com.cognite",
   organizationName := "Cognite",
   organizationHomepage := Some(url("https://cognite.com")),
-  version := "2.5.2",
+  version := "2.5.3",
```

Comment:
If you haven't publish new version with Nexus I think it's fine to just drop the previous build and publish the new one from this